e2e/v2_curl_test.go

```diff
@@ -139,10 +139,10 @@ func cURLPrefixArgs(clus *etcdProcessCluster, method string, req cURLReq) []stri
 		if clus.cfg.clientTLS != clientTLSAndNonTLS {
 			panic("should not use cURLPrefixArgsUseTLS when serving only TLS or non-TLS")
 		}
-		cmdArgs = append(cmdArgs, "--cacert", caPath, "--cert", certPath, "--key", privateKeyPath)
+		cmdArgs = append(cmdArgs, "--cacert", caPath, "--cert", pkcs12Path+":123", "--cert-type", "P12")
```

Comment:
password; I tried making it with a blank password but curl still complained asking for one